recipes/cython_gsl/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: cythongsl
+  version: {{ version }}
+
+source:
+  fn: cythongsl-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/C/CythonGSL/CythonGSL-{{ version }}.tar.gz
+  sha256: 177537590442be3ff0e21fb121cc3775a5dbf96611eeade317eee48d49282bd7
+
+build:
+  number: 0
+  skip: True  # [win]
+
+script: python setup.py build install --single-version-externally-managed --record record.txt
```

Comment:
Please remove the "build." I noticed I had some `copy-and-pasta` in my message above.